pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -214,3 +262,31 @@ func (ct *connTrackOvsCtl) GetMaxConnections() (int, error) {
 	}
 	return maxConns, nil
 }
+
+func statusStringToStateflag(status string) uint32 {
```

Comment:
could you add a link to the online spec where this mapping is defined?

also, not for this PR, but would it make more sense for the `StatusFlag` field in the `Connection` to be a string with the connection status rather than the numerical flag (for debugging purposes at least, assuming we don't use that field for anything)?

Reply:
Sure, added. 
Yes, I don't find any use of `StatusFlag` now. @srikartati, what do you think of changing its type to string?

Reply:
Yes ultimately we will send status as ipfix field and that will most likely be a string-- as there seems to be no standard IANA field for this. We could maintain it as a string however mapping would be needed if we are dumping flows in through conntrack_linux.go.